package-lock.json

```diff
@@ -5364,22 +5364,6 @@
         "url": "https://github.com/sponsors/sindresorhus"
       }
     },
-    "node_modules/@quasar/app-webpack/node_modules/globby/node_modules/fast-glob": {
```

Comment:
Any changes to this file with no changes to package.json is very worrying. Do not merge this without getting rid of it.